app/policies/application_policy.rb

```diff
@@ -2,52 +2,17 @@ class ApplicationPolicy
   attr_reader :user, :record
 
   def initialize(user, record)
+    raise Pundit::NotAuthorizedError, "must be logged in" unless user
     @user = user
     @record = record
   end
-
-  def index?
-    false
-  end
-
-  def show?
-    scope.where(:id => record.id).exists?
-  end
-
-  def create?
-    false
-  end
-
-  def new?
-    create?
-  end
-
-  def update?
-    user.present?
-  end
-
-  def edit?
-    update?
-  end
-
-  def destroy?
-    false
+  
+  def is_base?
+    @user.role? :base
   end
```

Comment:
Review the `base` role, I don't think this is meant for this app as we have the standard, admin, and premium roles only.